### PR TITLE
[READY]Da mafia update. Mob nicknames, new fedoras, suits!

### PR DESCRIPTION
**Pull request by Krysonism**
## About The Pull Request

This PR adds a new contraband crate to cargo; the Cosa Nostra starter pack.
The crate contains four complete mafia outfits, four pictures of the virgin Mary, 0-4 Italian moustaches and 10% chance of a single switchblade.

Burning a picture of the Madonna, inducts you into the mafia, letting you pick a mafia nickname that is inserted between your first and last name.

Lorenzo Robusto -> Lorenzo "The Carp" Robusto

The PR resprites the black fedora and white suit, makes the checkering on the checkered suit stronger and adds three new clothing items: the beige suit, beige fedora and white fedora.

The type of outfits you get from the crates are randomized about 60% of the outfits are the black version, about 15% are white and 15% are beige.

Thanks to @Qustinnus for helping me with this PR.

mafia_comp

## Why It's Good For The Game
Mafiatide has always been one of my favourite gimmicks and has been part of many of my best SS13  experiences. 
This PR adds some needed support to this gimmick by providing a money sink, a convenient way to get enough outfits and a way to recruit and keep track of the members of your crime family.

I've also improved the selection of suits and fedoras which can be reused for other purposes. The white suit and the fedora in particular look a lot better in my opinion.

## Changelog
:cl:
add: The Cosa Nostra starter pack  has been added to cargo as a contraband crate.
add: Added pictures of the virgin mary, burning one of these will let you pick a mafia nickname.
add: The beige suit, beige fedora and white fedora clothing items have been added to the game.
imageadd: The non-job fedoras and the white suit have been resprited.
/:cl:

### CODE REVIEW

**Comment by Krysonism** [comment]
picturemadonna64

Here is the Madonna icon. 

I am thinking of making one spawn inside the chaplain bible if he choose Catholicism as his religion.

**Comment by Twaticus** [comment]
"Now as that card burns, so may your soul burn in hell if you betray your friends in the family"

**Comment by Krysonism** [comment]
> 
> 
> "Now as that card burns, so may your soul burn in hell if you betray your friends in the family"

That sounds better. I just copypasted the oath from wikipedia.

File: code/game/objects/items/miscellaneous.dm [hunk 1]
```diff
@@ -145,3 +145,60 @@
...
+		mob_mobs += joe
+		joe.say("As burns this saint, so will burn my soul. I enter alive and I will have to get out dead.", forced = /obj/item/virgin_mary)
+		to_chat(joe, "<span class='userdanger'>Being inducted into the mafia does not grant antagonist status.</span>")
+	else
+		return ..()
+
+/obj/item/virgin_mary/suicide_act(mob/living/user)
+	user.visible_message("<span class='suicide'>[user] starts saying their Hail Mary's at a terrifying pace! It looks like [user.p_theyre()] trying to enter the afterlife!</span>")
+	user.say("Hail Mary, full of grace, the Lord is with thee. Blessed are thou amongst women, and blessed is the fruit of thy womb, Jesus. Holy Mary, mother of God, pray for us sinners, now and at the hour of our death. Amen. ", forced = /obj/item/virgin_mary)
+	sleep(50) //I can't get the timers to work, so sleep it is.
```
Comment (tralezab) [inline code/game/objects/items/miscellaneous.dm]: i'll help you with timers but you must use them, a 5 second sleep is a lot

**Comment by Ohagi-Chan** [comment]
Just noticed a Hail Mary suicide on that mary icon.
What does it look like, I just see the prayer?

File: code/game/objects/items/miscellaneous.dm (hunk 1 above)
Comment (Krysonism) [reply code/game/objects/items/miscellaneous.dm]: Thank you.

**Comment by Farquaar** [comment]
I just hope the nicknames don't cause any policy confusion for new players.

**Comment by Shadowflame909** [comment]
Being in the mafia should allow you to jump the ~~shark~~ Megacarp.

**Comment by cacogen** [comment]
your sprites are always really good

**Comment by Anonmare** [comment]
You're gonna talk a long walk out of a short airlock if you snitch

**Comment by Grazyn** [comment]
The beige hat kinda makes him look like he's wearing a headband on a dick-shaped head.

**Comment by FantasticFwoosh** [comment]
Just need brickphone, dirty greased up hair, big shades, gold chain and picture perfect Mafia defence lawyer.

always count on Ken Rosenberg

It'd still be nice to have a policy message to establish that being invited probably doesn't absolve you from crimes that someone antag-valid in the Mafia gang can commit. Though probably you could surround your gang with blood brothers and radio contacted traitors entirely for max criminality.

File: code/game/objects/items/miscellaneous.dm [hunk 2]
```diff
@@ -145,3 +145,60 @@
...
+			return ..()
+		if(!isliving(user) || !user.mind) //A sentient mob needs to be burning it, ya cheezit.
+			return ..()
+		var/mob/living/joe = user
+
+		if(joe in mob_mobs) //Only one nickname fuckhead
+			to_chat(joe, "<span class='warning'>You have already been initiated into the mafioso life.</span>")
+			return FALSE
+
+		to_chat(joe, "<span class='notice'>As you burn the picture you feel like you can pick up a fitting nickname</span>")
```
Comment (PKPenguin321) [inline code/game/objects/items/miscellaneous.dm]: Needs punctuation

**Review by PKPenguin321 — CHANGES_REQUESTED** [review]

File: code/game/objects/items/miscellaneous.dm (hunk 2 above)
Comment (ShizCalev) [inline code/game/objects/items/miscellaneous.dm]: . at the end.

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Comment by ShizCalev** [comment]
i'm disappointed by the lack of changes to italian_replacement.json

**Comment by Krysonism** [comment]
> i'm disappointed by the lack of changes to italian_replacement.json

I have some ideas for lines.

File: code/game/objects/items/miscellaneous.dm (hunk 2 above)
Comment (PKPenguin321) [reply code/game/objects/items/miscellaneous.dm]: and maybe a comma in the middle or something
"As you burn the picture you feel like you can pick up a fitting nickname" is really awkward

**Comment by Krysonism** [comment]
I will be fixing these issues once i get home from work.

Also frick huge dmis like head.dmi, I knew i would get conflicts. 

I am splitting out the fedoras and a few other proper hats into their own dmi.

File: code/game/objects/items/miscellaneous.dm (hunk 1 above)
Comment (Krysonism) [reply code/game/objects/items/miscellaneous.dm]: Fixed this

**Comment by ExcessiveUseOfCobblestone** [comment]
Headmins would prefer a pool ^_^

**Comment by Krysonism** [comment]
> 
> 
> Headmins would prefer a pool instead of custom input nicknames^_^

I think it would be a shame to force players to pick from a pool. For a RP focused feature like this allowing the players to express their creativity is a big part of the fun. 

n RP minded player who seeks out this feature is less likely to pick an infringing nickname, so i believe the administration burden will be quite low.

As seen in this list below, a lot of common styles of Mafia nicknames would not be able to be added to a prepicked list since they incorporate the persons first or last name in some fashion.

As the headmins initially seemed to be fine with the current setup i hope i can win them over.

https://en.wikipedia.org/wiki/List_of_Italian-American_mobsters

**Comment by ExcessiveUseOfCobblestone** [comment]
ok well just post a screencap or something when they okay it since this does touch names in a way that's otherwise not allowed per current naming policy.

**Comment by ExcessiveUseOfCobblestone** [comment]
gonna stale cause I need to see headmin approval of custom input (or something that says this  complies with naming policy) or said custom input gets removed: There's no point of reviewing this otherwise.

**Comment by Krysonism** [comment]
> 
> 
> gonna stale cause I need to see headmin approval of custom input (or something that says this complies with naming policy) or said custom input gets removed: There's no point of reviewing this otherwise.

I am in talks with the headmins.

**Comment by Krysonism** [comment]
I just got permission! Here is the screenshot as requested.

approv

I will resolve the dmi conflicts today.

File: code/game/objects/items/miscellaneous.dm [hunk 3]
```diff
@@ -145,3 +145,62 @@
...
+			new_name = "[copytext(joe.real_name, 1, space_position)] \"[nickname]\" [copytext(joe.real_name, space_position)]"
+		else //Append otherwise
+			new_name = "[joe.real_name] \"[nickname]\""
+		joe.real_name = new_name
+		used_up = TRUE
+		mob_mobs += joe
+		joe.say("As burns this saint, so will burn my soul. I enter alive and I will have to get out dead.", forced = /obj/item/virgin_mary)
+		to_chat(joe, "<span class='userdanger'>Being inducted into the mafia does not grant antagonist status.</span>")
+	else
+		return ..()
```
Comment (ExcessiveUseOfCobblestone) [inline code/game/objects/items/miscellaneous.dm]: if(!ignition_message)
    return ..()
code here

Comment (ExcessiveUseOfCobblestone) [reply code/game/objects/items/miscellaneous.dm]: you need to call inheritance here too for signaling purposes

**Comment by Iamgoofball** [comment]
please roll these guys into Families when it's merged

**Pull request merged**